### PR TITLE
Redesign: transparent background helper

### LINKED ISSUE #6535: Redesign: Transparent background
## Feature Description
Currently, various places are using variations of a transparent background with different square sizes. E.g. color picker, saved color, color input color preview.
The same background should be used everywhere.

See [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235).

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Summary
Adds helper to use consistent transparent background.

## User-facing changes
| Before | After |
|--------|------|
| Screenshot 2021-03-22 at 19 00 15 | Screenshot 2021-03-22 at 18 59 03 |

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add a shape and add semi-transparent background color to it.
2. Save the color to Saved colors.
3. Verify the transparent background looks similar in the following places:
- Saved colors
- Color picker (use gradient mode to test that) -- for the opacity line, gradient line, and gradient stops.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.
## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6535

### CODE REVIEW

**Comment by miina** [comment]
Hm, looks like the background of `Alpha` comes from the library... Will need to investigate if and how to replace that:
Screenshot 2021-03-22 at 17 13 10

File: assets/src/edit-story/components/colorPicker/currentColorPicker.js [hunk 1]
```diff
@@ -70,8 +70,16 @@ const HueWrapper = styled.div`
 `;
 
 const AlphaWrapper = styled.div`
-  background: #fff;
   ${wrapperCSS}
+  div:first-child div:first-child div:first-child {
```
Comment (miina) [inline assets/src/edit-story/components/colorPicker/currentColorPicker.js:74]: @barklund It looks like it's not really possible to overwrite the default opacity line background in a clean way -- this does it but it's not nice really. An alternative would be doing a custom opacity (doesn't feel like worth it just for the transparent background, another alternative would be revisiting the transparent bg with UX. Thoughts?

Comment (barklund) [reply assets/src/edit-story/components/colorPicker/currentColorPicker.js:74]: There's always gonna be conflicts with third-party components, so this works for me.

**Review by barklund — APPROVED** [review]
🌽

**Comment by miina** [comment]
Accidentally merged it 🤦‍♀️ Will let Claudio know.

**Pull request merged**